packages/front-end/services/DefinitionsContext.tsx

```diff
@@ -61,6 +67,10 @@ type DefinitionContextValue = Definitions & {
   getFactMetricById: (id: string) => null | FactMetricInterface;
   getExperimentMetricById: (id: string) => null | ExperimentMetricInterface;
   getMetricGroupById: (id: string) => null | MetricGroupInterface;
+  getTemplateById: (id: string) => null | ExperimentTemplateInterface;
+  getTemplateExperimentsById: (
```

Comment:
Remove this from definitions. The only thing in definitions should be the templates themselves, not the experiment mappings.